conf/fuzz.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2012-2020 The NATS Authors
```

Comment:
Sorry we missed that earlier. Since this is a new file, we usually put only the year the file is created, so that would be 2020 here.